pkg/registry/remote.go

```diff
@@ -44,7 +46,15 @@ func readRemoteImage(n image.Name) (v1.Image, error) {
 		return nil, err
 	}
 
-	return remote.Image(ref, remote.WithAuth(auth))
+	img, err := remote.Image(ref, remote.WithAuth(auth))
+	if err != nil {
```

Comment:
I think the daemon should be consulted _before_ the remote repository, otherwise there's a risk that a local update will be overlooked.

Reply:
Fixed.